docs/TuxMutableClass.md

```diff
@@ -0,0 +1,142 @@
+        <li><a href="#mutableClass">OBJECT: mutableClass</a>
+          <ol style="list-style-type: lower-alpha;">
+            <li><a href="#mutableClass-componentWillMount">METHOD: mutableClass.componentWillMount</a></li>
+            <li><a href="#mutableClass-shouldComponentUpdate">METHOD: mutableClass.shouldComponentUpdate</a></li>
+          </ol>
+        </li>
+      </ol>
+    </ol>
+  </li>
+  <li><a href="#TuxMutableClass-Guidelines">TuxMutableClass Guidelines</a></li>
+  <li><a href="#TuxMutableClass-Complete-Example">TuxMutableClass Complete Example</a></li>
+</ol>
+
+## <a id="Premise"></a>Premise [#](#Premise)
+>The Tux Mutable Class is an opinionated type of React class designed compare specified mutable traits to determine if a component should re-render.
+
+## <a id="Implementation"></a>Implementation [#](#Implementation)
+Tux determines if a mutable class component should re-render by taking advantage of the `componentWillMount` and `shouldComponentUpdate` life cycle events. Specifically in the `componentWillMount` life cycle, the component will do a deep search of its `state` and `props` to create paths to the specified `mutableTraits`. The paths are then attached to the constructor's prototype, such that a deep search to create the paths only needs to be done once for a particular type of component.
+
+These `mutableTraits` paths are then used in the `shouldComponentUpdate` life cycle event to see if those properties have changed when comparing the current state and props to the next props and state of the component. The `shouldComponentUpdate` life cycle event determines if a component should re-render in the virtual DOM. By performing this check you can prevent the unneeded re-rendering of components within the virtual DOM each time potential re-render is triggered.
+
+The mutable class in `Tux` is an extension of the owneeClass [***link here***](tuxedojs.org/docs/TuxReact/TuxOwneeClass).
+
+### <a id="Requiring-TuxMutableClass"></a>1) Requiring Tux Mutable Class [#](#Requiring Tux Mutable Class)
+The Tux Mutable Class method `createMutableClass` is exposed by requiring the main Tux module, which can be required via:
+
+```javascript
+  var React = require('tux');
+```
+
+**Note that at this time for Tux to be compiled by the JSX transformer, the Tux module must be assigned to the variable name `React`**
+
+### <a id="creating-mutable-class-components"></a>2) Creating Mutable Class Components [#](#creating-mutable-class-components)
+The Tux extended `React` object allows users to create mutable class components via invoking [React.createMutableClass](#createMutableClass). Pass in an object to the `createMutableClass` method and it will return a new object that has been extended with `componentWillMount` and `shouldComponentUpdate` React life cycle methods. The `createMutableClass` method is just like the `React.createClass` method, except it has extended `mutableClass` functionality.
```

Comment:
Change second sentence to ... method and it will return a `TuxMutableClass` which is an extension of a generic React Class.

Drop everything after that.